project/profiles/capacity4more/modules/c4m/user/c4m_user/c4m_user.module

```diff
@@ -1124,3 +1124,133 @@ function c4m_user_ckeditor_settings_alter(&$settings, $conf) {
     unset($settings['loadPlugins']['media']);
   }
 }
+
+/**
+ * Implements hook_entity_insert().
+ *
+ * Flags OG content that require notification, when commenting on existing
+ * content, and when new new content is added.
+ */
+function c4m_user_entity_insert($entity, $type) {
+  if (!og_context()) {
+    // Flagging is done for OG context only.
+    return;
+  }
+
+  if ($type == 'node') {
```

Comment:
@ordavidil 
We have this issue both for hook_entity_insert() and hook_entity_update().
This logic is fine as long as both insert and update perform exactly the same actions.
However, in our case, insert and update will perform different actions, when we add  'c4m follow comment' flag.
It might get even more complicated, when more flags are added.